test1-prod.js

```diff
@@ -3,3 +3,10 @@
 if(fooProd  === "bar") {
   console.log( 'hey' )
 }
+
+
+var fooProd2 = '' ;
+
+if(fooProd2  === "bar2") {
+  console.log( 'hey2' );
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/machadoit-test-org/eslint-test-prod/pullRequest?prid=6947672)
```suggestion
  console.log( "hey2" );
```